fusedrug/utils/file_formats/fasta_efficient_lookup_hdf5_index.py

```diff
@@ -0,0 +1,72 @@
+import h5py
+from Bio import SeqIO
+from typing import List
+
+
+class LargeFASTAConverter:
```

Comment:
Since you open a handle to a file on every call, it's better to change it to static utility functions.

Reply:
can you elaborate ?

Reply:
ahhh - I think I understand

Reply:
done :)